bottlecap/src/config/mod.rs

```diff
@@ -109,7 +108,6 @@ pub fn get_config(config_directory: &Path) -> Result<Config, ConfigError> {
     // TODO(duncanista): revert to using datadog.yaml when we have a proper serializer
     if path.exists() {
         log_failover_reason("datadog_yaml");
-        debug!("datadog.yaml is not supported, use environment variables instead");
```

Comment:
Also this, because the logging subsystem is not enabled, so this would never appear